src/main/java/net/marshmallow/BetterRecipeBook/Mixins/Centered/MoveEffects.java

```diff
@@ -32,7 +32,7 @@ private int moveEffectsRight(int i) {
             if (MinecraftClient.getInstance().player.isCreative()) {
                 return i + 323;
             } else {
-                return i + 304;
+                return i + 323;
```

Comment:
Why are we doing this?

Reply:
Admittedly, it was a lazy number change. With Inventorio installed, the width of the inventorio inventory is exactly the same as that of the Creative inventory.

Reply:
Same goes for the change in RemoveRecipeBookOffset. The book widget touches the edge of the inventorio which it should be slightly offset. Without the change here in MoveEffects, potion effects overlap the inventorio GUI